v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToSpanner.java

```diff
@@ -294,7 +300,20 @@ public interface Options extends PipelineOptions, StreamingOptions {
     @Default.Boolean(false)
     Boolean getRoundJsonDecimals();
 
-    void setRoundJsonDecimals(Boolean value);
+    @TemplateParameter.Text(
```

Comment:
For jsons, especially like these that can have upto 1000 keys, we read them from a gcs file. There is also a limit on the text type (not sure how much). Since we can have alot of shards it might be safer to read this from gcs. Also makes it easier to run it via gcloud.

We already do this for getSessionFilePath() template param.